k8s/examples/wordpress/wordpress/20-statefulset.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: wordpress
+spec:
+  selector:
+    matchLabels:
+      app: wordpress
+      tier: frontend
+  serviceName: wordpress
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: wordpress
+        tier: frontend
+    spec:
+      serviceAccountName: wordpress
+      containers:
+      - name: wordpress
+        image: wordpress:5.2-apache
+        env:
+        - name: WORDPRESS_DB_HOST
+          value: wordpress-mysql
+        - name: WORDPRESS_DB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: wordpress
+              key: wordpress-db-password
+        ports:
+        - name: wordpress
+          containerPort: 80
+        volumeMounts:
+        - name: wordpress-data
+          mountPath: /var/www/html
+  volumeClaimTemplates:
+  - metadata:
+      name: wordpress-data
+      labels:
+        storageos.com/replicas: "1"
```

Comment:
For reasons know to no-one labels on VCTs are only added to the actual PVC that gets created if the labels are part of the spec.selector.matchLabels as well as spec.template.metadata.labels